templates/injector-deployment.yaml

```diff
@@ -113,7 +113,7 @@ spec:
           readinessProbe:
             httpGet:
               path: /health/ready
-              port: 8080
+              port: {{ printf ":%v" .Values.injector.port | default ":8080" }}
```

Comment:
The `| default ":8080"` is actually not needed since the default is in the `values.yaml` file. Can you remove it?

Reply:
sorry for the copy/past mistake, done